easybuild/easyconfigs/p/PRIMUS/PRIMUS-1.9.0.eb

```diff
@@ -0,0 +1,25 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author: Guilherme Balzana
+# Research Computing Services - University of Cambridge
+
+easyblock = 'Tarball'
+
+name = 'PRIMUS'
+version = '1.9.0'
+
+homepage = 'https://primus.gs.washington.edu/primusweb/res/form.html'
+description = """PRIMUS: Pedigree Reconstruction and Identification of a
+    Maximum Unrelated Set
+"""""
+
+toolchain = SYSTEM
+
+source_urls = ['https://primus.gs.washington.edu/docroot/versions/']
+sources = ['PRIMUS_v%(version)s.tgz']
+checksums = ['89f8c76c43534ed9bdcaebc8c32bce4049120cacc661f2e215d223344b9f3b58']
+
+sanity_check_paths = {
+    'files': ["bin/run_PRIMUS.pl"],
```

Comment:
So, I think it's best to move this into GCCcore and depend on a perl version here.

Reply:
thanks @Micket for all the feedback.
this is my first time contributing to easyconfigs. 

by moving this into GCCcore do you mean setting this to the latest as well, right?

Reply:
@guimaluf Not that using GCCcore as toolchain also implies renaming the easyconfig file (that's why the tests fail).

Filename should be `PRIMUS-1.9.0-GCCcore-10.2.0.eb`, and you should also include `Perl` as a dependency (to prevent relying on `perl` provided by the OS).